package-lock.json

```diff
@@ -12,14 +12,14 @@
         "@astrojs/mdx": "^2.1.0",
         "@astrojs/tailwind": "^5.1.0",
         "@fontsource/ubuntu": "^5.0.8",
+        "@iconify-json/mdi": "^1.2.2",
```

Comment:
Weird, I didn't install anything actually. Maybe CodeSandbox messed up this when initiallizing the dev container? 🤔 @alexnm